freezeyt_blog/meeting9.md

```diff
@@ -0,0 +1,217 @@
+# Tests Reorganization and Parametrization
+
+**[9th Online Meeting](https://www.youtube.com/watch?v=JQMvylmLs2Q&list=PLFt-PM7J_H3EU5Oez3ZSVjY5pZJttP2lT&index=10&t=0s&ab_channel=encukou), August 31, 2020**
+
+***Mentor: Petr Viktorin***
+
+### Checking New PRs
+- Blog 6, change some images into code snippets.
+- Test_parse_absolute_url, add tests for netloc, port and path.
+
+*00hr:18mins*
+
+Petr started to reorganize tests on a separate branch because it was not to be done in one shot. In order not to loose the work done he commited it as WIP (work in progress) without pushing it.
```

Comment:
```suggestion
Petr started to reorganize tests on a separate branch because it was not to be done in one shot. In order not to lose the work done he commited it as WIP (work in progress) without pushing it.
```